Source/GameEngineVisuals/Program.cs

```diff
@@ -1,15 +1,590 @@
+        // start cordinate index from grid mid to allow for subtraction. ie (iLenght * 0,5).
+        public static void loadMapFromFile(MapVertex mapVertex)
+        {
+
+            string sContent = File.ReadAllText("Map.bsp"); ;
+            char[] cContent = sContent.ToCharArray();
+            int iOffset = 0;
+            //int iPath;
+
+            int i = 0; // mapVertex.m_iX / 2;   // 5, start at mid
+            int j = mapVertex.m_iY / 2;   // 5, start at mid
+
+            //int i = 5;   // 5, start at mid
+            //int j = 5;   // 5, start at mid
+
+            for (int x = 0; x != sContent.Length; x++)
+            {
+
+                switch (cContent[x])
+                {
+
+                    case 'D':
+                        //iPath = direction.down;
+
+                        for (int g = 0; g != iOffset; g++)
+                        {
+                            MapVertex vertex = new MapVertex(i, j);
+                            g_vPosition.Add(vertex);
+                            i++;
+
+                        }
+                        break;
+
+                    case 'U':
+                        for (int g = 0; g != iOffset; g++)
+                        {
+                            MapVertex vertex = new MapVertex(i, j);
+                            g_vPosition.Add(vertex);
+                            i--;
+
+                        }
+                        break;
+
+                    case 'L':
+                        //iPath = direction.left;
+                        for (int g = 0; g != iOffset; g++)
+                        {
+                            MapVertex vertex = new MapVertex(i, j);
+                            g_vPosition.Add(vertex);
+                            j--;
+
+                        }
+                        break;
+
+                    case 'R':
+                        for (int g = 0; g != iOffset; g++)
+                        {
+                            MapVertex vertex = new MapVertex(i, j);
+                            g_vPosition.Add(vertex);
+                            j++;
+
+                        }
+                        break;
+
+                    default:                                 // content was a number, ignore that and restart main while loop to check next char.
+                        iOffset = (int)Char.GetNumericValue(cContent[x]);
+
+                        //(Convert char to int)
+                        continue;
+                        //-----------------------------------------------------------------------------		
+
+                }
+
+
+            }
+        }
+
+
+
+
+
+
+
+
+
+
+        //-----------------------------------------------------------------------------
+        // Initilize Frame Buffer, 
+        //		fills the matrix with specifed symbol, default is 32
+        //-----------------------------------------------------------------------------
```

Comment:
I think it's better to either use xml comments (`///` + `Enter`) or multi-line comments. Xml comments provide descriptive information about a method for example and Intellisense can even extract information from the xml comment to show information in a tooltip (description, return value, param values etc).